res/values/strings.xml

```diff
@@ -120,6 +121,9 @@
     <string name="contact">Contact</string>
     <string name="verified_contact">Verified contact</string>
     <string name="camera">Camera</string>
+    <string name="camera_capture_description">Capture</string>
+    <string name="camera_change_description">Change camera</string>
```

Comment:
```suggestion
    <string name="camera_change_description">Switch camera</string>
```

Reply:
and can we just name the keys `capture` and `switch_camera`? although there are strings following other approaches, this is what we are doing usually as default in the last time, only if that turns out not to be sufficient, we add more information to the key.